src/mz/tests/e2e.rs

```diff
@@ -518,6 +518,16 @@ mod tests {
 
         assert!(output.trim().starts_with("Healthy: \tyes"));
 
+        let binding = cmd()
+            .arg("region")
+            .arg("show")
+            .arg("--region aws/us-east-1")
```

Comment:
Doesn't this need to be two separate `arg` calls? Or `--region=aws/us-east-1` (note equals sign).